src/translations/indexMapper.js

```diff
@@ -0,0 +1,362 @@
+import { arrayFilter, arrayMap } from './../helpers/array';
+import { getListWithRemovedItems, getListWithInsertedItems } from './maps/utils/visuallyIndexed';
+import { rangeEach } from '../helpers/number';
+import IndexMap from './maps/indexMap';
+import SkipMap from './maps/skipMap';
+import MapCollection from './mapCollection';
+import localHooks from '../mixins/localHooks';
+import { mixin } from '../helpers/object';
+import { isDefined } from '../helpers/mixed';
+
+class IndexMapper {
```

Comment:
Missing jsdoc about the purpose of this class

Reply:
Imagine a new person in the team reads this description:

> Index mapper manages the mappings (from index to another value) provided by "smaller" maps called index map(s).

I think this tells too little. An explanation about the purpose of the class would be more helpful:

> Index mapper is the centralised source of truth regarding row and column indexes. It is used to perform CRUD operations on indexes such as insert, move and remove, as awell as translating the physical indexes to visual indexes.

Reply:
@wszymanski Please keep comments line no wider than 120 chars or lower.

Reply:
Does it seem that `IndexMapper` is not typed in TS? The `columnIndexMapper` and `rowIndexMapper` are public APIs as public properties use that class. TS needs to be updated.